### PR TITLE
[grafana] Add envValueFrom to image-renderer

### PR DESCRIPTION
**Pull request by Red-M**
This allows using secrets or configmaps in-cluster to configure options for the image-renderer.
This is useful for setting the renderer token via an in-cluster secret.
From grafana/grafana-image-renderer#420

### CODE REVIEW

**Review by zanhsieh — CHANGES_REQUESTED** [review]
@Red-M 
Thank you for the contribution. Would you mind to bump the version in Chart.yaml to latest so the CI can merge this PR once approved?

**Comment by Red-M** [comment]
Done

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**
